src/clients/dotnet/TigerBeetle/UInt128Extensions.cs

```diff
@@ -101,3 +102,65 @@ public static UInt128 ToUInt128(this BigInteger value)
         }
     }
 }
+
+/// <summary>
+/// Safe generation of Universally Unique and Byte-Sortable Identifiers as UInt128s.
+/// </summary>
+public static class ID
+{
+    private static long idLastTimestamp = 0L;
+    private static readonly byte[] idLastRandom = new byte[12];
```

Comment:
Should it be 10 bytes?
```suggestion
    private static readonly byte[] idLastRandom = new byte[10];
```